libs/errorHandler.js

```diff
@@ -3,18 +3,21 @@
 /*  Load gulp + plugins
 /* ------------------------------------------------------------------------- */
 
-var notify  = require("gulp-notify");
+var gutil = require('gulp-util');
```

Comment:
We were using gulp-util already. It seems this error handler is formatting the output only and not handling other functions yet. Did you happen to find gulp-util more useful than gulp-notify?